package.json

```diff
@@ -17,53 +17,53 @@
     "csurf": "1.9.0",
     "envc": "2.5.0",
     "escape-html": "1.0.3",
-    "express": "4.15.2",
+    "express": "4.15.3",
     "jpm": "1.3.1",
     "keygrip": "1.0.1",
     "mobile-detect": "1.3.6",
-    "morgan": "1.8.1",
-    "mozlog": "2.1.0",
+    "morgan": "1.8.2",
+    "mozlog": "2.1.1",
     "node-statsd": "0.1.1",
     "nodemon": "1.11.0",
     "nodify-uuid": "0.0.1",
-    "pg": "6.1.5",
+    "pg": "6.2.3",
     "pg-patcher": "0.3.0",
-    "raven": "1.2.1",
-    "raven-js": "3.14.2",
+    "raven": "2.0.2",
```

Comment:
Noticed this warning message in the [changelog](https://github.com/getsentry/raven-node/blob/master/History.md#200---5102017) for raven:

> Be aware when upgrading: it is possible that with raven-node 2.0, your node process will shut down from exceptions where it previously did not

Looking at the server code, I don't think this should affect us, but thought I'd mention it anyway